pandas/tests/test_nanops.py

```diff
@@ -339,6 +339,13 @@ def test_nanmean_overflow(self):
                 self.assertEqual(result, np_result)
                 self.assertTrue(result.dtype == np.float64)
 
+    def test_nanstd_roundoff(self):
+        # Regression test for GH 10242 (test data taken from GH 10489). Ensure
+        # that variance is stable.
+        data = Series(766897346 * np.ones(10))
```

Comment:
can u add a couple of more tests with some fixed values and a fixed result (that is not 0)

also included an example with nans